python/ray/data/dataset.py

```diff
@@ -1116,7 +1116,7 @@ def randomize_block_order(
 
         Returns:
             The block-shuffled :class:`Dataset`.
-        """
+        """  # noqa: E501
```

Comment:
Why didn't we need these `noqa` before?

Reply:
lint doesn't run check on the entire code base, so my guess is that no-one touches these part before except for me now

Reply:
Oh, had no idea. Surprised we hadn't checked this earlier (especially since we frequently change `dataset.py`)